sql/mysql/internal/mysqlversion/mysqlversion.go

```diff
@@ -84,6 +84,13 @@ func (v V) SupportsIndexComment() bool {
 	return v.Maria() || v.GTE("5.5.3")
 }
 
+// SupportsUUIDDataType reports if the version supports uuid
+// data types, as from maria db 10.7.0 they are supported and offer
+// better performance over the varchar(32) and equal to binary(16) solutions.
+func (v V) SupportsUUIDDataType() bool {
+	return v.Maria() && v.GTE("10.7.0")
+}
+
```

Comment:
```suggestion
```

This method is not used anymore.

Reply:
ah missed that one removed it :+1:

Reply:
Hey @michaelboke, I think you forgot to push the local changes because I still see this function.

Reply:
@michaelboke: remind, to release this PR. :D

Reply:
@a8m @giautm thanks for the notice!, i now really pushed the changes to GH 😄